docs/source/bentocloud/how-tos/manage-models-and-bentos.rst

```diff
@@ -1,97 +1,54 @@
-===========================
+========================
 Manage Models and Bentos
-===========================
+========================
 
-In this guide, we'll walk through the process of managing your Models and Bentos
-effectively in BentoML. This includes pulling and pushing Models and Bentos,
-dealing with new versions, and viewing their status.
+BentoML provides both a local Model Store and a Bento Store to host your models and Bentos, respectively. Once you've pushed these entities to BentoCloud,
```

Comment:
I don't think so. Models here just refer to machine learning models. They are used everywhere in the AI industry...Definitely not a term in the BentoML ecosystem, so "models" will do.

Reply:
Understood. I was just thinking that
models -> ML models in general
Models -> BentoML models because we do have `bentoml.save_model` and `bentoml.load_model` which makes Models actually a special noun for us, since it's one of the unit of distribution and we do have our way of packaging a model

Reply:
I see what you mean. But perhaps let's just keep it simple for "models" or users may even want to think about this for some time when reading it. I don't think there is a big difference in its meaning. You can say "load the model" to imply that the entity result of `bentoml.load_model`, but in essense, it comes from an ML model.